### DISCUSSION TITLE
"Precondition Failed": what is it?

### DISCUSSION THREAD

**Discussion by chlohr** [fwupd/fwupd#6719] [General]
Hello,
  I'm trying to update my PC. I get an error "Precondition Failed". I don't know how to handle this. How to know more about the precondition in question?

# fwupdmgr update
Downloading 0.1.27.0 for OptiPlex 7060 System Firmware...
Fetching firmware https://fwupd.org/downloads/0ad1e4377c480eeaff32c8a0ef3d580a18c823564bc351cc4899becd2a324cb7-firmware_Dell_OptiPlex_7060_BIOS_1.27.0.Wu.cab

Failed to download https://fwupd.org/downloads/0ad1e4377c480eeaff32c8a0ef3d580a18c823564bc351cc4899becd2a324cb7-firmware_Dell_OptiPlex_7060_BIOS_1.27.0.Wu.cab: Precondition Failed

(Note: this is fwupd version 1.2.14 on Debian 10.13, current LTS release)

Best regards

**Comment by hughsie** [comment]
1.2.x is too old I'm afraid. See https://lore.kernel.org/lvfs-announce/CAD2FfiGty1wtcjaRMZXaW8LAEQ_-mchOeEAtKN48kX_O_i5_NQ@mail.gmail.com/T/#t for more details. You probably need to need to ask the debian-lts team to upgrade the fwupd package.

**Comment by davidjao** [comment]
As a temporary workaround, you could just download the .cab file manually using the clickable link in your comment and then run `fwupdmgr` manually:

    fwupdmgr --allow-older --allow-reinstall install <file.cab>

Obviously in this case you're responsible if stuff breaks.
